FlowCrypt/Common/Services/Local Storage/LocalStorage.swift

```diff
@@ -16,7 +16,7 @@ protocol LocalStorageType {
 struct LocalStorage: LocalStorageType {
 
     private enum Constants: String, CaseIterable {
-        case indexCurrentUser = "indexCurrentUser"
+        case indexCurrentUser = "keyCurrentUser"
```

Comment:
Some other day we may want to migrate this too, to `indexCurrentUser`. Because our app does encryption, "key" is often used for private information then for storage access keys (indexes). Therefore it will be better to call these "index" to remove ambiguity.

But for this time, we'll leave it as is to keep things simple, as we agreed.

Reply:
Better to create a ticket for that migration. If we just change a key here it will log out all users.

Reply:
I understand, will make a ticket